scripts/common.mk

```diff
@@ -0,0 +1,11 @@
+# Makefile script used in various OP-TEE test components
```

Comment:
License/copyright

Reply:
Wasn't in original file from optee_test, but added/